lib/facter/mongodb_version.rb

```diff
@@ -2,7 +2,7 @@
   setcode do
     if Facter::Core::Execution.which('mongo')
       mongodb_version = Facter::Core::Execution.execute('mongo --version 2>&1')
-      %r{^MongoDB shell version:?\s+([\w\.]+)}.match(mongodb_version)[1]
+      %r{^MongoDB shell version(: | v)([\w\.]+)}.match(mongodb_version)[2]
```

Comment:
I don't think this should be needed

Reply:
I'm happy to remove that but I'm unable to test as I don't have access to that cluster any more.